static/_redirects

```diff
@@ -9,3 +9,5 @@
 # Other redirects
 # Github page with information about Antrea "live" office hours
 /live https://github.com/jayunit100/k8sprototypes/blob/master/antrea-LIVE/README.md 200
+
+https://charts.antrea.io/* https://antrea.io/charts/:splat
```

Comment:
The path doesn't exist in "Deploy Preview", is it expected? https://deploy-preview-25--gracious-lamport-f690a0.netlify.app/charts

Reply:
It's https://deploy-preview-25--gracious-lamport-f690a0.netlify.app/charts/index.yaml that matters. There is no index.html file in the `charts/` directory, hence the error. But helm will try to retrieve index.yaml.

Reply:
got it